### ISSUE TITLE
Grid multiselect is broken

### ISSUE THREAD

**Issue by marcushellberg** [vaadin/flow#3558]
When I set Grid in multiselect mode, I expect it to work.

- Keyboard navigation/selection does not work
- Rows remain highlighted even though not selected. 
- Select all checkbox does not affect any row checkboxes

<img width="974" alt="screen shot 2018-02-14 at 7 14 29 pm" src="https://user-images.githubusercontent.com/883698/36239032-839a5436-11bb-11e8-86b7-7819883c9ee0.png">

Platform alpha18

**Comment by pleku** [comment]
Select all not working related to: https://github.com/vaadin/vaadin-grid-flow/issues/73
Keyboard selection not working related to:  https://github.com/vaadin/vaadin-grid-flow/issues/74

Let's verify this issue once those are fixed.

**Comment by pleku** [comment]
The select-all issue has been fixed, while the keyboard selection issue is still open.

I still feel that this issue is probably outdated and also in the wrong place (not in Grid repo), so I'm closing this one. Any issues related to Grid should go to vaadin/vaadin-grid-flow.
